cli/lib/fetch.js

```diff
@@ -9,22 +9,19 @@ const fetch = require('node-fetch');
 const https = require('https');
 
 const httpAgent = new http.Agent({
-    family: 6, hints: dns.ADDRCONFIG | dns.V4MAPPED
-})
+  family: 6,
```

Comment:
Node docs are really sparse on this, but `4` and `6` are actually used for the IP address family (IPv4 and IPv6). 

It would be nice if there were a constant for this:

```javascript
const IP_FAMILIES: {
  V4: 4,
  V6: 6
}
```
without being explicit, it looks like a ["magic number"](https://en.wikipedia.org/wiki/Magic_number_(programming)) to me.